gleam.toml

```diff
@@ -11,8 +11,8 @@ links = [
 ]
 
 [dependencies]
-gleam_stdlib = ">= 0.34.0 and < 2.0.0"
-gleam_json = ">= 1.0.0 and < 2.0.0"
+gleam_stdlib = ">= 0.44.0 and < 2.0.0"
```

Comment:
This doesn't upgrade anything! The upper bound is still v2. Undo this change please 🙏